daemon/indexing/listener/listener.js

```diff
@@ -1,7 +1,7 @@
 const fs = require('fs')
 const http = require('http')
 const urllib = require('url')
-const Origin = require('origin')
+const Origin = require('../../../dist/index')
```

Comment:
I think @tomlinton updated the origin-js container to generate a local origin npm package. so if you update origin-box and recreate your origin-js container, you should not need to change this ?

Reply:
Ahh yes this should not be in, was using it for local testing. Removing this change...

Reply:
Thanks for catching this, removed...